discovery-infra/download_logs.py

```diff
@@ -102,6 +112,12 @@ def get_ui_url_from_api_url(api_url: str):
         raise KeyError(api_url)
 
 
+def download_must_gather(kubeconfig: str, dest_dir: str):
+    log.info(f"Downloading must-gather to {dest_dir}")
+    command = f"oc --insecure-skip-tls-verify --kubeconfig={kubeconfig} adm must-gather --dest-dir {dest_dir} > {dest_dir}/must-gather.log"
```

Comment:
one more time oc command? Please talk to @RazRegev and use oc utils

Reply:
Yeah